pkg/test/framework/runtime/components/mixer/native.go

```diff
@@ -124,7 +132,11 @@ func (c *nativeComponent) Start(ctx context.Instance, scope lifecycle.Scope) (er
 		return err
 	}
 
-	c.client.attributeManifest, err = deployment.ExtractAttributeManifest(c.client.workdir)
+	helmExtractDir, err := ctx.CreateTmpDirectory("helm-mixer-attribute-extract")
```

Comment:
@ozevren - This was causing Mixer to parse helm spew as if it were mixer config with bad effects

Reply:
Was that due to folder location only?

Reply:
Yup, the helm stuff was written into the dir that is then treated as the root of the mixer config